content/coop.html

```diff
@@ -311,23 +312,22 @@ <h3> Gender </h3>
   Women are paid less than men on average for co-op terms.
   </span>
 
-  Although Women are paid less than men, this is not necessarily related to a gender bias. Some
-  possible reasons could be different interests in terms of work location, type of work, etc among
+  Although women are paid less than men, this is not necessarily related to a gender bias. Some
+  possible reasons could be due to different interests in terms of work location, type of work, etc. among
   men and women respondents in the class.
 
-  NOTE: there were about 42-30 men and 9-8 women who responded to coop salaries
+  NOTE: there were about 42-30 men and 9-8 women who responded to coop salaries.
```

Comment:
why is this 42-30 and 9-8 instead of 30-42 and 8-9?

Reply:
i'll take a look; this was the original wording, so i hadn't questioned it haha

Reply:
Since the graph provides percent I just wanted to clarify that the responses for each term and gender varied between 42 to 30 men and 9 to 8 women